examples/hello-world/hello-flower/jobs/hello-flwr-pt/app/custom/pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "flwr_pt"
+version = "1.0.0"
+description = ""
+license = "Apache-2.0"
+dependencies = [
+    "flwr[simulation]>=1.11.0,<2.0",
+    "nvflare~=2.5.0rc",
+    "torch==2.2.1",
+    "torchvision==0.17.1",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["."]
+
+[tool.flwr.app]
+publisher = "nvidia"
+
+[tool.flwr.app.components]
+serverapp = "flwr_pt.server:app"
+clientapp = "flwr_pt.client:app"
+
+[tool.flwr.app.config]
+num-server-rounds = 3
+
+[tool.flwr.federations]
+default = "local-simulation"
+
+[tool.flwr.federations.local-simulation]
+options.num-supernodes = 2
```

Comment:
we might need to add in the documentation (or have links to flower documentation) to explain how these mean or what things can user configure in this "pyproject.toml"

Reply:
Yes, will be added once flower has the docs.